docs/src/prototypes/dropdowns/MentionsWithDropdown.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import keyboardKey from 'keyboard-key'
+import { Dropdown, DropdownProps } from '@stardust-ui/react'
+
+import { atMentionItems } from './dataMocks'
+import { insertTextAtCursorPosition } from './utils'
+import { PortalAtCursorPosition } from './PortalAtCursorPosition'
+
+interface MentionsWithDropdownState {
+  dropdownOpen?: boolean
+  searchQuery?: string
+}
+
+const editorStyle: React.CSSProperties = {
+  backgroundColor: 'lightgrey',
+  padding: '5px',
+  minHeight: '100px',
+  outline: 0,
+}
+
+class MentionsWithDropdown extends React.Component<{}, MentionsWithDropdownState> {
```

Comment:
this is just about visual representation - frankly, it was not absolutely evident to me that this grey box below is introduced as an _editable area_. I do understand that this was done to keep styles (and amount of code) minimal - and totally support that 💯 But maybe we could improve a bit in this aspect, by doing the following steps:

- change wording of description a bit to explicitly mention that area is editable
- use brighter styles that would differentiate area from regular rectangle

![image](https://user-images.githubusercontent.com/9024564/53237116-ef5f9300-36a6-11e9-8c6e-039bedd7ff03.png)


Please, feel free to decide on whether changes should be done, and which ones specifically. Let me just, for the sake of reference, share changes that I have tried on my side to achieve this appearance:

```js
const editorStyle: React.CSSProperties = {
  backgroundColor: '#eee',
  padding: '5px',
  minHeight: '100px',
  outline: 0,
  border: '1px dashed grey',
  borderRadius: '5px'
}
```

Reply:
I like it, applied the text change and style on content editable div, thanks!